sktime/utils/plotting.py

```diff
@@ -91,6 +121,20 @@ def plot_series(*series, labels=None, markers=None):
 
         plot_func(x=x, y=y, ax=ax, marker=marker, label=label, color=color)
 
+    # plot prediction intervals if present
+    if pred_int is not None:
+        check_pred_int(pred_int)
+        # check same conditions as for earlier indices
+        if all([x in index for x in pred_int.index]):
```

Comment:
What happens if this condition is not true?

Reply:
We need to make sure that the function runs reliable and either produces a plot as expected or returns an informative error to the user.

Reply:
@Ifeanyi30 please take a look at these comments here

Reply:
Ok, the first comment is for the action 'if pred_int is present' while the second is for the code block that compares the pred_int and series indices.

Reply:
Yes, so what happens if the statement is not true? I believe nothing happens, no error is raised and no prediction intervals will be plotted. Is that correct?